service/routes.py

```diff
@@ -184,25 +182,34 @@ def post(self):
         Creates a Order
         This endpoint will create a Order based the data in the body that is posted
         """
-        pass
+        app.logger.info("Request to create a customer order")
+        check_content_type("application/json")
+        order = CustomerOrder()
+        order.deserialize(request.get_json())
```

Comment:
Should we use api.payload instead of request.get_json()? Then line 188 becomes order.deserialize(api.payload)

Reply:
It seems like the corresponding unit test would fail otherwise. Maybe we could remove those tests?